bin/jobs_functions.js

```diff
@@ -114,6 +122,36 @@ function terminateTestJob(testjobId) {
         })
 }
 
+function deleteTestJob(testjobId) {
+    let url = config.server + '/api/v1/testjobs'+ '/' + testjobId + '/delete';
+    util.output('Deleting job: '+testjobId);
+    request.delete(url)
+        .auth(config.username, config.password)
+        .accept('application/junit+xml')
+        .send()
+        .end((err, result) => {
+            if (err !== null) {
+                if (('status' in err) && ('message' in result.body)) {
+                    switch (err['status']) {
+                        case 403:
+                            util.error(err['status'] + ': ' + result.body['message']);
```

Comment:
This one is does the same as the default-case. So it can be removed. Maybe we dont need the switch at all?